cli/crates/cli/tests/dev_watch.rs

```diff
@@ -45,4 +45,10 @@ fn dev_watch() {
     assert!(user_id.starts_with("user_"));
     assert!(user_birthday.ends_with('Z'));
     assert_eq!(user_verified, "VERIFIED");
+
+    {
+        // Check that the TS resolver types are being generated.
+        let generated_types_path = env.directory.join("grafbase/generated/index.ts");
```

Comment:
I think they're different because the schemas are in the "dot grafbase" directory, `~/.grafbase`, whereas this is inside the grafbase directory in the project. I don't think we would want it to be in `~/.grafbase` because that would make it trickier to import. It's also part of the project and some people will want to commit it.

Reply:
Do you think this place where we now store the resolvers could also be used for storing the client types? Could these files collide?

Reply:
Hmm, that's a good question. Maybe `./grafbase/resolver/types.ts` kind of path? We would only have to be careful with the gitignores but maybe it makes sense.